src/sentry/conf/server.py

```diff
@@ -1506,18 +1506,233 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
+        "schedule": task_crontab("*/10", "*", "*", "*", "*"),
+    },
+    "dynamic-sampling-boost-low-volume-transactions": {
+        "task": "telemetry-experience:sentry.dynamic_sampling.tasks.boost_low_volume_transactions",
+        "schedule": task_crontab("*/10", "*", "*", "*", "*"),
+    },
+    "dynamic-sampling-recalibrate-orgs": {
+        "task": "telemetry-experience:sentry.dynamic_sampling.tasks.recalibrate_orgs",
+        "schedule": task_crontab("*/10", "*", "*", "*", "*"),
+    },
+    "dynamic-sampling-sliding-window-org": {
+        "task": "telemetry-experience:sentry.dynamic_sampling.tasks.sliding_window_org",
+        "schedule": task_crontab("*/10", "*", "*", "*", "*"),
+    },
+    "custom_rule_notifications": {
+        "task": "telemetry-experience:sentry.dynamic_sampling.tasks.custom_rule_notifications",
+        "schedule": task_crontab("*/10", "*", "*", "*", "*"),
+    },
+    "clean_custom_rule_notifications": {
+        "task": "telemetry-experience:sentry.dynamic_sampling.tasks.clean_custom_rule_notifications",
+        "schedule": task_crontab("*/7", "*", "*", "*", "*"),
+    },
+    "weekly-escalating-forecast": {
+        "task": "issues:sentry.tasks.weekly_escalating_forecast.run_escalating_forecast",
+        "schedule": task_crontab("0", "0", "*", "*", "*"),
+    },
+    "schedule_auto_transition_to_ongoing": {
+        "task": "issues:sentry.tasks.schedule_auto_transition_to_ongoing",
+        "schedule": task_crontab("*/5", "*", "*", "*", "*"),
+    },
+    "github_comment_reactions": {
+        "task": "integrations:sentry.integrations.github.tasks.github_comment_reactions",
+        "schedule": task_crontab("0", "16", "*", "*", "*"),
+    },
+    "statistical-detectors-detect-regressions": {
+        "task": "performance:sentry.tasks.statistical_detectors.run_detection",
+        "schedule": task_crontab("0", "*/1", "*", "*", "*"),
+    },
+    "refresh-artifact-bundles-in-use": {
+        "task": "attachments:sentry.debug_files.tasks.refresh_artifact_bundles_in_use",
+        "schedule": task_crontab("*/1", "*", "*", "*", "*"),
+    },
+    "on-demand-metrics-schedule-on-demand-check": {
+        "task": "performance:sentry.tasks.on_demand_metrics.schedule_on_demand_check",
+        "schedule": task_crontab("*/5", "*", "*", "*", "*"),
+    },
+    "uptime-detection-scheduler": {
+        "task": "uptime:sentry.uptime.detectors.tasks.schedule_detections",
+        "schedule": task_crontab("*/1", "*", "*", "*", "*"),
+    },
+    "demo_mode_sync_debug_artifacts": {
+        "task": "demomode:sentry.demo_mode.tasks.sync_debug_artifacts",
+        "schedule": task_crontab("0", "*/1", "*", "*", "*"),
+    },
+    "relocation-find-transfer-region": {
+        "task": "relocation:sentry.relocation.transfer.find_relocation_transfer_region",
+        "schedule": task_crontab("*/5", "*", "*", "*", "*"),
+    },
     "sync_options_trial": {
```

Comment:
We could remove this now that we're ready to start using the task scheduler for real work.